scripts/generate-config-data.js

```diff
@@ -240,7 +240,7 @@ function augment (data, extra) {
   return dataAugmentedOrdered;
 }
 
-function generateAugmentedData (source) {
+function generateAugmentedData (source, extra) {
```

Comment:
This function is exported outside of this module as `getData`, most likely we need to pass `extra` there as well

Reply:
yep, missed that, fixed